app/views/admin/splits/_details.html.erb

```diff
@@ -0,0 +1,35 @@
+<article class="card">
+   <div class="ArticleHeader">
+      <h2 class="top">Details</h2>
+    </div>
+    <table class="borderless bottom">
+        <tr class="population-row">
+            <td>Population Size</td>
+            <td>
+                <span class='population'><%= split.assignments.count %></span>
+                <span>(<%= link_to "Upload New Assignments", new_admin_split_bulk_assignment_path(split) %>)</span>
+            </td>
+        </tr>
+        <tr>
+            <td>Created At</td>
+            <td><%= split.created_at %></td>
+        </tr>
+        <tr>
+            <td>Finished?</td>
+            <td><%= split.finished? %></td>
+        </tr>
+        <tr>
+            <td>Owner App</td>
+            <td><%= split.owner_app.name %></td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>
+                <div class="split-decision">
+                    <span class="flag">🏁</span>
+                    <%= link_to "Decide Split", new_admin_split_decision_path(split) %>
+                </div>
+            </td>
+        </tr>
+    </table>
+</article>
```

Comment:
Ah you're right, html files here have 4 spaces throughout. The "ArticleHeader" addition was my own, which follows 2 space-spacing. 

Does it make sense to tackle the discrepancy in spacing throughout in this PR or in the layout-revamp-change? Also, should we follow 2 or 4 spaces? 

(Of course ensuring this file at least is consistent for this PR).